src/components/Build.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+export default function Build (props){
+    let boardStyle = {
+        backgroundColor:"#EAD87A",
+        height:"100vh",
+        width:"100vh",
+        position:"fixed",
+        left:"500px",
+        top: props.top
```

Comment:
I think using `position: sticky` here instead of `fixed` will make it easier to integrate this component with everything else, since `sticky` elements aren't removed from the normal flow of the document, meaning they take up space (whereas with `fixed`, it would overlap elements around it). Also, assuming that you decided to put the `top` property in props to make integrating it with other components easier, I think `left` should also be in props?